platform_versioned_docs/version-24.1.1/enterprise/kubernetes.mdx

```diff
@@ -14,6 +14,8 @@ Complete the following procedures to install Seqera Platform Enterprise on a Kub
 
 ### Create a namespace
 
+Seqera uses a dedicated namespace to isolate Kubernetes pods and services from the other resources on your cluster.
```

Comment:
```suggestion
Create a namespace to isolate Kubernetes resources used Seqera Platform from the other resources on your cluster.
```

Reply:
Think we should avoid the proliferation of `-nf` suffix and prefix. Seqera Platform is beyond nextflow